packages/addons/service/multimedia/boblightd/patches/boblightd-2.0.5-add_aml_client.patch

```diff
@@ -0,0 +1,462 @@
+--- a/src/Makefile.am	2014-01-25 23:48:24.000000000 +0100
++++ b/src/Makefile.am	2014-12-08 21:11:50.586778992 +0100
+@@ -7,6 +7,7 @@
+ 	 -g
+ 
+ bin_PROGRAMS = boblightd \
++	boblight-aml \
+ 	boblight-constant
+         
+ 
+@@ -19,6 +20,11 @@
+ 
+ endif
+ 
++boblight_aml_SOURCES  = clients/boblight-aml/boblight-aml.cpp
++boblight_aml_SOURCES += clients/boblight-aml/flagmanager-aml.cpp
++boblight_aml_SOURCES += clients/flagmanager.cpp
++boblight_aml_SOURCES += util/misc.cpp
++
+ boblight_v4l_SOURCES = \
+ 	clients/boblight-v4l/boblight-v4l.cpp \
+ 	clients/boblight-v4l/flagmanager-v4l.cpp \
+diff -urPp src/clients/boblight-aml.cpp src/clients/boblight-aml/boblight-aml.cpp
+--- a/src/clients/boblight-aml.cpp	1970-01-01 01:00:00.000000000 +0100
```

Comment:
build fails because file created in src/clients not src/clients/boblight-aml. bad c/p ?

should be --- /dev/null

same for flagmanager-aml.cpp and flagmanager-aml.h

or pick http://sprunge.us/SfDj